### PR TITLE
feature/add-edit-hotspot-screen

### PR DESCRIPTION
**Pull request by sebi-dan-csi**
https://user-images.githubusercontent.com/72596583/154693728-fe15ce41-7a3c-402b-9e9e-34e91c9d9a01.mp4

### CODE REVIEW

File: components/Appbar.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export const Appbar = () => {
     <PaperAppbar
       style={{
         paddingTop: top,
-        backgroundColor: theme.colors.background,
+        backgroundColor: 'white',
```
Comment (mveres) [inline components/Appbar.tsx:29]: let's stick to theme colors only

File: components/InputField.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  KeyboardTypeOptions,
+  ReturnKeyTypeOptions,
+  StyleProp,
+  StyleSheet,
+  Text,
```
Comment (mveres) [inline components/InputField.tsx:7]: let's use `Text` from react native paper

File: components/InputField.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  multiline?: boolean;
+  editable?: boolean;
+  keyboardType?: KeyboardTypeOptions | undefined;
+};
+
+export const InputField = (props: InputFieldProps) => {
+  const theme = useTheme();
+  const styles = StyleSheet.create({
+    title: {
+      color: theme.colors.text,
```
Comment (mveres) [inline components/InputField.tsx:32]: this won't be needed if we use react native paper typography

File: screens/AddHotspotScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,414 @@
+import { capitalize } from 'lodash';
+import React, { useState } from 'react';
+import {
+  Dimensions,
+  Image,
+  ScrollView,
+  StyleSheet,
+  Text,
```
Comment (mveres) [inline screens/AddHotspotScreen.tsx:8]: same here, use from paper

File: screens/AddHotspotScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+import SelectDropdown from 'react-native-select-dropdown';
+import catLadyImage from '../assets/cat-lady2.png';
+import currentLocationIcon from '../assets/current-location.png';
+import mapPinIcon from '../assets/map-pin.png';
+import { Appbar } from '../components/Appbar';
+import { InputField } from '../components/InputField';
+import { HotspotStatus } from '../models/Hotspot';
+import { Location } from '../models/Location';
+
+('react-native-paper-dropdown');
```
Comment (mveres) [inline screens/AddHotspotScreen.tsx:29]: what is this good for?

**Review by mveres — COMMENTED** [review]
export a better resolution for the map-pin png, and other future assets

File: components/Appbar.tsx (hunk 1 above)
Comment (sebi-dan-csi) [reply components/Appbar.tsx:29]: In figma, the screen background is white. I should add 'background' to theme colors as '#FFF' in order to override it.

Comment (mveres) [reply components/Appbar.tsx:29]: Then figma needs to fix the discrepancy between the defined palette and the used colors.
We'll raise this issue to UX, meanwhile let's stick to the theme defined colors. No hardcoding off other colors.

File: screens/AddHotspotScreen.tsx (hunk 5 above)
Comment (sebi-dan-csi) [reply screens/AddHotspotScreen.tsx:29]: Not needed anymore. I've tried before a different library for dropdowns, and its import remained there. I removed it.

File: components/Appbar.tsx (hunk 1 above)
Comment (sebi-dan-csi) [reply components/Appbar.tsx:29]: Ok. I've switched back to `theme.colors.background`.

**Comment by sebi-dan-csi** [comment]
> export a better resolution for the map-pin png, and other future assets

At least map-pin is exported at 4x.

File: screens/AddHotspotScreen.tsx (hunk 4 above)
Comment (sebi-dan-csi) [reply screens/AddHotspotScreen.tsx:8]: done.

**Review by mveres — COMMENTED** [review]
👍

**Pull request merged**